src/main/java/core/basesyntax/Lottery.java

```diff
@@ -8,8 +8,8 @@ public Ball getRandomBall() {
         String color = colorSupplier.getRandomColor();
         Ball ball = new Ball();
         ball.setColor(color);
-        Random random = new Random();
-        ball.setNumber(random.nextInt(100));
+        int randomNumber = new Random().nextInt(100);
```

Comment:
a constant is a static final field that is used for better code understanding when you have some 'magic' numbers like 100, 66, random strings and so on

when you don't understand immediately why we have 100 here, and not another number